quarkus/admin/build.gradle.kts

```diff
@@ -84,3 +85,17 @@ tasks.named("sourcesJar") { dependsOn("compileQuarkusGeneratedSourcesJava") }
 tasks.named("javadoc") { dependsOn("jandex") }
 
 tasks.named("quarkusDependenciesBuild") { dependsOn("jandex") }
+
+tasks.named("distZip") { dependsOn("quarkusBuild") }
+
+tasks.named("distTar") { dependsOn("quarkusBuild") }
+
+distributions {
+  main {
+    contents {
+      from("../../NOTICE")
+      from("../../LICENSE-BINARY-DIST").rename("LICENSE-BINARY-DIST", "LICENSE")
+      from(project.layout.buildDirectory) { include("polaris-quarkus-admin-*-runner.jar") }
```

Comment:
nit: would it make sense to include shell start scripts (similar to Spark, for example)?

Reply:
Good idea, but we can't use the `startScript` task here (it doesn't understand Quarkus layouts). Let's tackle that in a follow-up task.